docs/actions.md

```diff
@@ -102,6 +102,16 @@ Executes the following steps:
 - uses: it-at-m/lhm_actions/action-templates/actions/action-checkout
 ```
 
+### action-filter
+
+[Path-Filter](https://github.com/dorny/paths-filter). Input is the filter. Output are the filtert paths.
```

Comment:
This doesn't fit to the other documentation:
- a sentence what this Action does
- an example with inputs and outputs
if it's just a wrapper around an existing Action you can link to the wrapped Action (instead of the example)

Reply:
```suggestion
[Path-Filter](https://github.com/dorny/paths-filter) GitHub Action hat enables conditional execution of workflow steps and jobs, based on the files modified by pull request, on a feature branch, or by the recently pushed commits.
```yml
- uses: it-at-m/lhm_actions/action-templates/actions/action-filter@main
  id: changes
  with:
    filters: |
      src:
        - 'src/**'

  # run only if some file in 'src' folder was changed
- if: steps.changes.outputs.src == 'true'
  run: ...
```
```